pants.ini

```diff
@@ -234,6 +234,9 @@ eslint_setupdir: %(pants_supportdir)s/eslint
 eslint_config: %(pants_supportdir)s/eslint/.eslintrc
 eslint_ignore: %(pants_supportdir)s/eslint/.eslintignore
 
+[native-toolchain]
+enable_libc_search: True
```

Comment:
If this is only for use in travis (which is maybe a bit odd?) then you could put it in `pants.travis-ci.ini`.

Reply:
It is, and it is a bit odd. I'll move it into that file.

Reply:
Moved in b902a7f, along with a more complete docstring describing why this is necessary.